README.md

```diff
@@ -34,11 +34,11 @@ from cuqi.problem import BayesianProblem
 # Step 1: Model and data, y = Ax
 A, y_data, info = Deconvolution2D.get_components(dim=128, phantom=grains())
 
-# Step 2: Prior, x ~ LMRF(0, 0.01)
-x = LMRF(location=np.zeros(A.domain_dim),
-                 scale=0.01,
-                 bc_type='neumann',
-                 physical_dim=2)
+# Step 2: Prior, x ~ LMRF(0,0.01)
+x = LMRF(location=0,
```

Comment:
In the updated, a scalar zero is given for location, in combination with the geometry, will zeros be filled for the location values of all the elements then? What is a numpy 1D array is given for the location and the size is not compatible with the geometry?  I understood from you that all distributions would now also allow passing a `dim` parameter, does this include XMRF, and if so how are the possibly conflicting ways of passing input handled?